wiki/Installation.md

```diff
@@ -60,4 +60,10 @@ pod 'DynamicWorkflow/UIKit'
 pod 'DynamicWorkflow/Core'
 ```
 
+#### In both of these cases you'll import Workflow as such
+
+```swift
+import Workflow
+```
```

Comment:
Is this the truth? I took a look at the podspec and I believe it is, but wanted another check